### ISSUE TITLE
Inconsistent cgroup name presentation on the UI

### ISSUE THREAD

**Issue by cakrit** [netdata/netdata#5663]
##### Bug report summary
The cgroup name on the right menu seems to be truncated some times. Not sure if there's a max length limit, but it looks strange. Everything works as expected, just need to see where that happens and record it here. 

##### Component Name
Probably web

##### Steps To Reproduce
Not sure, but it doesn't seem to have anything to do with the length of the name. Saw a much longer name getting presented properly. 

Details:

`/api/v1/charts`

```
"cgroup_k8s_kube-system_fluentd-gcp-v3.1.1-l87m5_d3636d06-440d-11e9-ad4e-42010a80013a_fluentd-gcp.cpu_limit": 		{
			"id": "cgroup_k8s_kube-system_fluentd-gcp-v3.1.1-l87m5_d3636d06-440d-11e9-ad4e-42010a80013a_fluentd-gcp.cpu_limit",
			"name": "cgroup_k8s_kube_system_fluentd_gcp_v3.1.1_l87m5_d3636d06_440d_11e9_ad4e_42010a80013a_fluentd_gcp.cpu_limit",
			"type": "cgroup_k8s_kube-system_fluentd-gcp-v3",
			"family": "cpu",
			"context": "cgroup.cpu_limit",
```

Screenshot with right menu showing both a full name cgroup and a truncated name cgroup (the one in the JSON above).

![Screenshot from 2019-03-18 14-11-35](https://user-images.githubusercontent.com/43294513/54532954-62e47f80-4989-11e9-8117-80c59ec03ea4.png)

And the logs from `cgroup-name.sh`:
```
2019-03-18 10:39:15: cgroup-name.sh: INFO: k8s containerID 'e281a09d8d4a5d4ae6abe4960b3bb92e2a7d1907da5c8d611b392af73adbb858' has chart name (namespace_podname_poduid_containername) 'k8s_kube-system_fluentd-gcp-v3.1.1-l87m5_d3636d06-440d-11e9-ad4e-42010a80013a_fluentd-gcp'
2019-03-18 10:39:15: cgroup-name.sh: INFO: cgroup 'kubepods_burstable_podd3636d06-440d-11e9-ad4e-42010a80013a_e281a09d8d4a5d4ae6abe4960b3bb92e2a7d1907da5c8d611b392af73adbb858' is called 'k8s_kube-system_fluentd-gcp-v3.1.1-l87m5_d3636d06-440d-11e9-ad4e-42010a80013a_fluentd-gcp'
```

**Comment by cakrit** [comment]
It's because of the "." in the chart name. Will need to check the Javascript and apply the split on the last dot only.

**Comment by ilyam8** [comment]
Fixed in #11050
